### PR TITLE
Prefer 'instrument' to 'form' in narrative

### PR DESCRIPTION
**Pull request by skadauke**
# Description

REDCap uses the terms "form" and "instrument" interchangeably. I believe that the word "instrument" has a more specific meaning than "form" and is also the word that data managers would use to describe what a REDCap instrument is. This PR changes "form" to "instrument" in most instances in the docs and outputs. 

### PR Checklist

Before submitting this PR, please check and verify below that the submission meets the below criteria:

- [x] Code changes are less than **250** lines total
- [x] The appropriate reviewer is assigned to the PR
- [x] The appropriate developers are assigned to the PR

# Code Review
This section to be used by the reviewer and developers during Code Review after PR submission

### Code Review Checklist

- [ ] I checked that new files follow naming conventions and are in the right place
- [ ] I checked that documentation is complete, clear, and without typos
- [ ] I added/edited comments to explain "why" not "how"
- [ ] I checked that all new variable and function names follow naming conventions
- [ ] I checked that new tests have been written for key business logic and/or bugs that this PR fixes
- [ ] I checked that new tests address important edge cases

### CODE REVIEW

File: R/import_redcap.R [hunk 1]
```diff
@@ -88,33 +89,33 @@ import_redcap <- function(redcap_uri,
...
-  # identifiers are kept regardless. This requires adding the form with
-  # identifiers to our redcap_read_oneshot call but filtering out extra,
-  # non-identifier fields in that form
+  # The user may not have requested instrument with identifiers. We need to
+  # make sure identifiers are kept regardless. This requires adding the
+  # instrument with identifiers to our redcap_read_oneshot call but filtering
+  # out extra, non-identifier fields in that instrument
 
-  # Set default forms for API call
+  # Set default instruments for API call
```
Comment (rsh52) [inline R/import_redcap.R:97]: Hm, we're changing the comment but leaving the variable below it as `forms_for_api_call`. Should we be changing those as well? It will have a trickle out effect into other variables/functions (like `check_forms_exist` right below).

File: R/import_redcap.R [hunk 2]
```diff
@@ -319,7 +320,7 @@ add_metadata <- function(supertbl, db_metadata, redcap_uri, token) {
     ) %>%
     relocate("field_name", "field_label", "field_type", .before = everything())
 
-  ## Create a record with the project identifier for each form
+  ## Create a record with the project identifier for each instrument
   all_forms <- unique(db_metadata$redcap_form_name)
```
Comment (rsh52) [inline R/import_redcap.R:324]: See above comment.

File: R/labelled.R [hunk 3]
```diff
@@ -97,9 +97,13 @@ make_labelled <- function(supertbl, format_labels = NULL) {
...
     redcap_event = "REDCap Event",
     redcap_arm = "REDCap Arm",
+    # TODO: `survey_timestamp` is not a predefined label but it constructed
+    # from [instrument_name]_timestamp. We can either a) change universally
+    # to "survey_timestamp" (which is what we do with `form_status_complete`)
+    # or else we need to change the below.
     survey_timestamp = "REDCap Survey Timestamp",
     redcap_survey_identifier = "REDCap Survey Identifier",
-    form_status_complete = "REDCap Form Status"
+    form_status_complete = "REDCap Instrument Completed?"
```
Comment (rsh52) [inline R/labelled.R:106]: For the sake of covering all bases, we want to keep this variable as `form_status_complete` and `instrument_status_complete`?

File: tests/testthat/test-labelled.R [hunk 4]
```diff
@@ -151,7 +151,7 @@ test_that("make_labelled applies all predefined labeles", {
     redcap_arm = "REDCap Arm",
     survey_timestamp = "REDCap Survey Timestamp",
     redcap_survey_identifier = "REDCap Survey Identifier",
-    form_status_complete = "REDCap Form Status"
+    form_status_complete = "REDCap Instrument Completed?"
```
Comment (rsh52) [inline tests/testthat/test-labelled.R:154]: See above comment about `form/instrument_status_complete`

**Review by rsh52 — CHANGES_REQUESTED** [review]
Generally looks good, just want to be sure we're clarifying this change everywhere it should be implemented. There are some variables/functions where this could result in crossed wires.

File: R/import_redcap.R (hunk 1 above)
Comment (skadauke) [reply R/import_redcap.R:97]: I think the rule should be to use "form" whenever you're programming and interacting with the REDCap API, and use "instrument" whenever you're writing human readable text.

File: R/labelled.R (hunk 3 above)
Comment (skadauke) [reply R/labelled.R:106]: See above

File: R/import_redcap.R (hunk 1 above)
Comment (rsh52) [reply R/import_redcap.R:97]: Sounds good then, in this one case it's a little confusing in just this instance then since the comment and the variable are next to each other.

**Review by rsh52 — APPROVED** [review]
